src/api.js

```diff
@@ -0,0 +1,12 @@
+const BASE_URL = 'https://bootcamp-api.codeit.kr/api/sample/'
+
+export async function getUserInfo(path) {
+  const response = await fetch(BASE_URL + path);
+
+  if (!response.ok) {
```

Comment:
response 가 아예 없는 경우를 대비해서 안전하게 ? 추가해주시는 거 추천드립니다!
```suggestion
  if (!response?.ok) {
```